v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/options/KafkaToBQFlexOptions.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.options;
+
+import com.google.cloud.teleport.metadata.TemplateParameter;
+import com.google.cloud.teleport.v2.kafka.options.KafkaReadOptions;
+import org.apache.beam.sdk.options.Default;
+
+/**
+ * The {@link KafkaToBQFlexOptions} class provides the custom execution options passed by the
+ * executor at the command-line.
+ */
+public interface KafkaToBQFlexOptions extends KafkaReadOptions, BigQueryStorageApiStreamingOptions {
```

Comment:
I'd call it KafkaToBigQueryFlexOptions (we usually don't shorten BigQuery to BQ, and even if shortened, it should be Bq instead of BQ as per the camel case style).

Reply:
Done